1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,46 @@
-function readingList(books) {
+const readingList = (books) => {
   // Write your code here...
-}
+  const mainDiv = document.querySelector("#content");
+  const bookList = document.createElement("ul");
+
+  let bookContent = books.map((book) => {
+    book.alreadyRead
+      ? (readOrNot = "book-item-not-read")
+      : (readOrNot = "book-item-read");
+
+    return `<li class=${readOrNot}>
```

Comment:
This definitely works (and is easier to read). We may be supposed to use the document.createElement() function and slowly create individual elements, and then piece them together as children and siblings, but like I said, this works!

Also, you should also apply some classnames to apply some better css (especially the image sizes)!